ntoskrnl/include/internal/ke_x.h

```diff
@@ -1359,6 +1359,7 @@ KxQueueReadyThread(IN PKTHREAD Thread,
 
     /* Sanity checks */
     ASSERT(Prcb == KeGetCurrentPrcb());
+    ASSERT(Prcb->PrcbLock != 0);
```

Comment:
How does this assert succeed on UP, where `KiAcquirePrcbLock` is a no-op?

Reply:
Oops, indeed. So it also needs to be placed in `CONFIG_SMP`. :sweat_smile:

Reply:
https://github.com/reactos/reactos/pull/6391